config/samples/infrastructure_v1alpha2_azuremachine.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+kind: AzureMachine
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: azuremachine-sample
+spec:
+  # Add fields here
+  foo: bar
```

Comment:
In the cluster-api, cluster-api-provider-aws, cluster-api-bootstrap-provider-kubeadm, and cluster-api-provider-docker repos we've started adding this directory to .gitignore, since they are autogenerated and invalid.